packages/compile-solidity/src/run.ts

```diff
@@ -171,7 +230,11 @@ function prepareCompilerInput({
  * @param sources - { [sourcePath]: string }
  * @return { [sourcePath]: { content: string } }
  */
-function prepareSources({ sources }) {
+function prepareSources({
+  sources
+}: {
+  sources: Common.Sources.Sources;
+}): PreparedSources {
```

Comment:
You might want to define `XOptions` interfaces for these args, as personally I find this syntax to be far less readable. There are a few other occurrences of this, but I'll just flag this one to cut down on the noise.

Reply:
sure, I don't mind